android-project/app/src/main/assets/roms/.gitignore

```diff
@@ -0,0 +1 @@
+*.zip
```

Comment:
Please don't litter potentially packaged directories with git metafiles.  Add a line to the top-level .gitignore file to suppress messages for this directory.